query/benchmarker_test.go

```diff
@@ -52,3 +56,206 @@ func TestProcessorHandler(t *testing.T) {
 		t.Errorf("total queries wrong: want %d got %d", qLimit, p1.count+p2.count)
 	}
 }
+
+func TestBenchmarkRunnerGetBufferedReaderPanicOnMissingFile(t *testing.T) {
+	dumbFileName := "some-random-file-that-should-not-exist"
+	_, err := os.Stat(dumbFileName)
+	if os.IsExist(err) {
+		t.Fatalf("File '%s' should not exist", dumbFileName)
+	}
+	b := &BenchmarkRunner{
+		fileName: dumbFileName,
+	}
+	defer func() {
+		if r := recover(); strings.HasPrefix(r.(string), "cannot open file for read") {
+			fmt.Println("Recovered in f", r)
```

Comment:
just to keep the output clean, can you make it only print if something goes wrong?

also most of these things should probably be lowercase to start, per Golang's error format